src/current/v25.1/set-up-logical-data-replication.md

```diff
@@ -85,87 +96,148 @@ When you run LDR in [`immediate` mode](#modes), you cannot replicate a table wit
+
+1. On cluster **B**, run:
+
+    {% include_cached copy-clipboard.html %}
+    ~~~ shell
+    cockroach encode-uri {user}:{password}@{node IP}:26257 --ca-cert {path to CA certificate} --inline
+    ~~~
+
+    The connection string output contains the source cluster's certificate:
+
+    ~~~
+    {user}:{password}@{node IP}:26257?options=-ccluster%3Dsystem&sslinline=true&sslmode=verify-full&sslrootcert=-----BEGIN+CERTIFICATE-----{encoded certificate}-----END+CERTIFICATE-----%0A
+    ~~~
+
+1. On cluster **A**, create an [external connection]({% link {{ page.version.version }}/create-external-connection.md %}) using cluster B's connection string (source in LDR stream 2). Prefix the `postgresql://` scheme to the connection string and replace `{source}` with your external connection name:
+
+    {% include_cached copy-clipboard.html %}
+    ~~~ sql
+    CREATE EXTERNAL CONNECTION {source} AS 'postgresql://{user}:{password}@{node IP}:26257?options=-ccluster%3Dsystem&sslinline=true&sslmode=verify-full&sslrootcert=-----BEGIN+CERTIFICATE-----{encoded certificate}-----END+CERTIFICATE-----%0A';
     ~~~
 
 ## Step 3. Start LDR
 
-In this step, you'll start the LDR job from the destination cluster. You can replicate one or multiple tables in a single LDR job. You cannot replicate system tables in LDR, which means that you must manually apply configurations and cluster settings, such as [row-level TTL]({% link {{ page.version.version }}/row-level-ttl.md %}) and user permissions on the destination cluster.
+In this step, you'll start the LDR stream(s) from the destination cluster. You can replicate one or multiple tables in a single LDR job. You cannot replicate system tables in LDR, which means that you must manually apply configurations and cluster settings, such as [row-level TTL]({% link {{ page.version.version }}/row-level-ttl.md %}) and user permissions on the destination cluster.
 
 <a id="modes"></a>_Modes_ determine how LDR replicates the data to the destination cluster. There are two modes:
 
 - `immediate` (default): {% include {{ page.version.version }}/ldr/immediate-description.md %}
 - `validated`: {% include {{ page.version.version }}/ldr/validated-description.md %}
 
-1. From the **destination** cluster, start LDR. Use the fully qualified table name for the source and destination tables:
+### Syntax
 
-    {% include_cached copy-clipboard.html %}
-    ~~~ sql
-    CREATE LOGICAL REPLICATION STREAM FROM TABLE {database.public.source_table_name} ON 'external://{source_external_connection}' INTO TABLE {database.public.destination_table_name};
-    ~~~
+There are two different SQL statements for starting an LDR stream, depending on your requirements:
```

Comment:
suggest "LDR streams can be started using one of the following SQL statements, depending ... "

Reply:
Nice, done!